Containers/talk/start.sh

```diff
@@ -16,43 +16,46 @@ elif [ -z "$INTERNAL_SECRET" ]; then
 fi
 
 set -x
-IPv4_ADDRESS_TALK="$(dig nextcloud-aio-talk A +short)"
+if [ -n "$(dig nextcloud-aio-talk A +short | grep -P "^[0-9.]+$" | head -n1)" ]; then
+    IPv4_ADDRESS_TALK="$(dig nextcloud-aio-talk A +short | grep -P "^[0-9.]+$" | head -n1)"
+fi
```

Comment:
this should always only return one ip-adress, no?

Reply:
Yes, currently can handle only one IPv6 and/or IPv4 address to advertise for TURN relays.

Reply:
the ipv4 ip to talk container (or nothing if no ipv4)

Reply:
What I meant was this:
```suggestion
IPv4_ADDRESS_TALK="$(dig nextcloud-aio-talk A +short)"
```

Reply:
It will always be just one ip and a ipv4 ip, no? (I think the check is not necessary here)

Reply:
I would keep it

Reply:
Why?

I dont understand how this command can return more than one value...

Reply:
I know unrealistic, but multiple docker networks

Reply:
But the talk container is by design only added to the nextcloud-aio network. How should this possibly happen?